README.md

```diff
@@ -87,29 +87,44 @@ To see a sample of what FlamingPy can do, let us first import a few important ob
 
 ```
 from flamingpy.codes import SurfaceCode
-from flamingpy.noise import CVLayer
+from flamingpy.cv.ops import CVLayer
```

Comment:
AFAIK this import is already updated to fit the current structure of main, this change would revert it.
```suggestion
from flamingpy.noise import CVLayer
```

Reply:
hey! humm, as far as I understood the CVLayer was moved to flamingpy.cv.ops and indeed it only works in that way

Reply:
Okay, I see - then there has been a misunderstanding. There might be confusion between the two versions:
1. PyPI release `v0.8.2a5`. In this version you should use `from flamingpy.cv.ops import CVLayer`, see the [corresponding documentation](https://flamingpy.readthedocs.io/en/v0.8.2a5/usage/getting_started.html).
2. Current `main` branch. This branch is continuously developed and can contain features that are not yet present in a release (we have monthly releases). In this version, you should use `from flamingpy.noise import CVLayer`.

You can simply commit my suggestion to fix it 🛠️

Reply:
Please see #57, in particular the changes to the README file